modules/administration/pages/image-management.adoc

```diff
@@ -228,29 +232,36 @@ Activation Keys ensure that images using a profile are assigned to the correct c
 .Relationship Between Activation Keys and Image Profiles
 [NOTE]
 ====
-When you associate an activation key with an image profile you are ensuring any image using the profile will use the correct software channel and any packages in the channel.
+When you associate an Activation Key with an image profile you are ensuring any image using the profile will use the correct software channel and any packages in the channel.
 ====
 
 . Click the btn:[Create] button.
 
 
 
 [[at.images.docker.sourceexample]]
-=== Example Dockerfile and add_packages Script
+==== Example of Dockerfile sources
 
 
-This section contains an example dockerfile.
-You specify a dockerfile that will be used during image building when creating an image profile.
-A dockerfile and any associated scripts should be stored within an external or internal Github or Gitlab repository.
+An Image Profile that can be reused is published at https://github.com/SUSE/manager-build-profiles
 
-The dockerfile provides access to a specific repository version served by {productname}.
-The following example dockerfile is used by {productname} to trigger a build job on a build host.
+[NOTE]
+.Important instructions in the Dockerfile
+====
 The [option]``ARG`` parameters ensure that the built image is associated with the desired repository version served by {productname}.
-The [option]``ARG`` parameters also allow you to build image versions of {sls} which may differ from the version of {sls} used by the build host itself.
+The [option]``ARG`` parameters also allow you to build image versions of {sls} which may differ from the version of {sls} used by the Build Host itself.
 
 For example: The [comand]``ARG repo`` parameter and the [command]``echo`` command pointing to the repository file, creates and then injects the correct path into the repository file for the desired channel version.
 
-_The repository version is determined by the activation key that you assigned to your image profile._
+_The repository version is determined by the Activation Key that you assigned to your image profile._
+====
+
+[NOTE]
+.Packages Required for Inspecting Your Images
+====
+To inspect images and provide the package and product list of a container to the {productname} {webui} you must install [package]#python# and [package]#python-xml# within the container.
+Without these packages your images will still build, but the package and product list will be unavailable from the {webui}.
```

Comment:
```suggestion
If you do not install them, images will still build but the package and product list will not available in the {webui}.
```

Reply:
It seems odd to me that we say "you must install these things" followed immediately by "but it'll work if you don't".